master/static/migrations/20220311095512_add_workspace_tables.up.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE workspaces (
+  id SERIAL PRIMARY KEY,
+  name TEXT UNIQUE NOT NULL,
+  archived BOOLEAN,
+  user_id INT REFERENCES users(id)
```

Comment:
**question:** does this need creation time?

Reply:
yes it does, I noticed when I was working on the 2nd part of this... I will add it to this migration to keep it cleaner